server/test/unit/server/db/test_connection.py

```diff
@@ -68,26 +82,30 @@ def tearDown(self):
     @patch('pulp.server.db.connection.mongoengine')
     def test__DATABASE_uses_default_name(self, mock_mongoengine):
         mock_mongoengine.connect.return_value.server_info.return_value = {'version': '2.6.0'}
+        host = 'champs.example.com:27018'
+        replica_set = ''
```

Comment:
I didn't expect replica_set to be set in this test. It's ok that it is, but couldn't replica_set get disincluded and line 94 would also get eliminated?